providers/okta/resources/organization.go

```diff
@@ -0,0 +1,180 @@
+	args["companyName"] = llx.StringData(settings.CompanyName)
+	args["status"] = llx.StringData(settings.Status)
+	args["subdomain"] = llx.StringData(settings.Subdomain)
+	args["address1"] = llx.StringData(settings.Address1)
+	args["address2"] = llx.StringData(settings.Address2)
+	args["city"] = llx.StringData(settings.City)
+	args["state"] = llx.StringData(settings.State)
+	args["phoneNumber"] = llx.StringData(settings.PhoneNumber)
+	args["postalCode"] = llx.StringData(settings.PostalCode)
+	args["country"] = llx.StringData(settings.Country)
+	args["supportPhoneNumber"] = llx.StringData(settings.SupportPhoneNumber)
+	args["website"] = llx.StringData(settings.Website)
+	args["endUserSupportHelpURL"] = llx.StringData(settings.EndUserSupportHelpURL)
+	args["created"] = llx.TimeDataPtr(settings.Created)
+	args["lastUpdated"] = llx.TimeDataPtr(settings.LastUpdated)
+	args["expiresAt"] = llx.TimeDataPtr(settings.ExpiresAt)
+
+	return args, nil, nil
+}
+
+func (o *mqlOktaOrganization) optOutCommunicationEmails() (bool, error) {
+	conn := o.MqlRuntime.Connection.(*connection.OktaConnection)
+	client := conn.Client()
+
+	ctx := context.Background()
+	settings, _, err := client.OrgSetting.GetOktaCommunicationSettings(ctx)
+	if err != nil {
+		return false, err
+	}
+
+	if settings.OptOutEmailUsers == nil {
+		return *settings.OptOutEmailUsers, nil
+	}
+
+	return false, nil
+}
+
+func (o *mqlOktaOrganization) billingContact() (*mqlOktaUser, error) {
+	conn := o.MqlRuntime.Connection.(*connection.OktaConnection)
+	client := conn.Client()
+
+	ctx := context.Background()
+	contactUser, _, err := client.OrgSetting.GetOrgContactUser(ctx, "BILLING")
+	if err != nil {
+		return nil, err
+	}
+	uid := contactUser.UserId
+
+	usr, _, err := client.User.GetUser(
+		ctx,
+		uid,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return newMqlOktaUser(o.MqlRuntime, usr)
+}
+
+func (o *mqlOktaOrganization) technicalContact() (*mqlOktaUser, error) {
+	conn := o.MqlRuntime.Connection.(*connection.OktaConnection)
+	client := conn.Client()
+
+	ctx := context.Background()
+	contactUser, _, err := client.OrgSetting.GetOrgContactUser(ctx, "TECHNICAL")
+	if err != nil {
+		return nil, err
+	}
+
+	uid := contactUser.UserId
+
+	usr, _, err := client.User.GetUser(
+		ctx,
+		uid,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return newMqlOktaUser(o.MqlRuntime, usr)
+}
+
+func (o *mqlOktaOrganization) securityNotificationEmails() (interface{}, error) {
+	conn := o.MqlRuntime.Connection.(*connection.OktaConnection)
+	client := conn.Client()
+
+	ctx := context.Background()
+	apiSupplement := &sdk.ApiExtension{
+		RequestExecutor: client.CloneRequestExecutor(),
+	}
+
+	emails, err := apiSupplement.GetSecurityNotificationEmails(
+		ctx,
+		conn.OrganizationID(),
+		conn.Token(),
+		ranger.DefaultHttpClient(),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return convert.JsonToDict(emails)
+}
+
+// GetThreatInsightSettings returns the Threat Insight settings for the organization
```

Comment:
nit: this should now be:
```suggestion
// threatInsightSettings returns the Threat Insight settings for the organization
```

Reply:
+1, since this is now internal i'm removing it